dashboard/v1.1/src/pages/Monitoring/Monitoring.tsx

```diff
@@ -80,54 +82,89 @@ const Monitoring: FC<MonitoringProps> = ({ updateLoader }) => {
   }, [updateLoader]);
   if (error) {
     return (
-      <Card>
-        <CardContent>
-          <h4>Monitoring</h4>
-          <hr />
-          <Alert severity="error">Unable to reach the service: error</Alert>
-        </CardContent>
-      </Card>
+      <>
+        <Card>
+          <CardContent>
+            <h4>Monitoring</h4>
+            <hr />
+            <Alert severity="error">Unable to reach the service: error</Alert>
+          </CardContent>
+        </Card>
+        <InputFab />
+      </>
     );
   }
   if (!response.data) {
     return (
-      <Card>
-        <CardContent>
-          <h4>Monitoring</h4>
-          <hr />
-          <Alert severity="info">Fetching data from sources</Alert>
-        </CardContent>
-      </Card>
+      <>
+        <Card>
+          <CardContent>
+            <h4>Monitoring</h4>
+            <hr />
+            <Alert severity="info">Fetching data from sources</Alert>
+          </CardContent>
+        </Card>
+        <InputFab />
+      </>
     );
   }
   updateLoader(false);
   return (
-    <Card>
-      <CardContent>
-        {!showRouteDetails || !routeDetailsData ? (
-          <>
-            <h4>
-              Monitoring{' '}
-              <ServicesState
-                active={(status: boolean) => setIsMonitoringActive(status)}
+    <>
+      <Card>
+        <CardContent>
+          {!showRouteDetails || !routeDetailsData ? (
+            <>
+              <h4>
+                Monitoring{' '}
+                <ServicesState
+                  active={(status: boolean) => setIsMonitoringActive(status)}
+                />
+              </h4>
+              <hr />
+              <Matrix
+                timeSeriesPath={response.data}
+                showRouteDetails={showDetails}
+                isMonitoringActive={isMonitoringActive}
               />
-            </h4>
-            <hr />
-            <Matrix
-              timeSeriesPath={response.data}
-              showRouteDetails={showDetails}
-              isMonitoringActive={isMonitoringActive}
+            </>
+          ) : (
+            <RouteDetailsComponent
+              routesChains={routeDetailsData}
+              showDetails={showDetails}
             />
-          </>
-        ) : (
-          <RouteDetailsComponent
-            routesChains={routeDetailsData}
-            showDetails={showDetails}
-          />
-        )}
-      </CardContent>
-    </Card>
+          )}
+        </CardContent>
+      </Card>
+      <InputFab />
+    </>
   );
 };
 
 export default Monitoring;
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Can you create a new file with the `InputFab` component?

Reply:
Sure @muskankhedia, I'll create a separate file. Also, should I delete the listItem from sidebar because as of now I've just commented it out ?

Reply:
Yes, you should delete the list item of `quick input` from the sidebar.